### ISSUE TITLE
MITRE WG: Establish alignment with CCC Cloud Taxonomy Working Group for capturing Controls Testing Requirements

### ISSUE THREAD

**Issue by nas-hub** [finos/common-cloud-controls#74]
### Assumptions
1. The CCC Cloud Taxonomy working group will be responsible for defining controls and their corresponding testing requirements.
1. The controls defined as part of the CCC Control Catalog will include clearly defined testing requirements.

### Objective
The objective is to establish a comprehensive set of Control Testing Requirements for each Control, enabling a thorough assessment of the effectiveness of these controls. The Controls Testing requirements will be articulated in plain English and translated into corresponding MITRE TTPs and abstract testing language like Gherkins.

**Comment by mcleo-d** [comment]
FYI - @mark-rushing - This items spans the `MITRE` and `Taxonomy` working groups in collaboration. I have added to your kanban to be tracked.

**Comment by eddie-knight** [comment]
Today's Taxonomy WG call determined that the group is happy to review and give feedback on a POC PR from the MITRE WG. The only expectation/requirement is that the output should live alongside the other service-level items in this repo.

@nas-hub is there anything else that needs to be discussed or done as part of this issue?
